CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+##  v1.11.0 (02/08/2020)
+
+**Common**
+
+* Added API to the `olp::cache::DefaultCache` to get or set the size of the mutable cache. Now you can increase or decrease the disk cache size in runtime (note: the elements are evicted).
```

Comment:
I would split this as it is also a getter for the protected cache size, not only for the mutable cache.

```suggestion
* Added API to the `olp::cache::DefaultCache` to get or set the size of the mutable cache. Now you can increase or decrease the disk cache size during runtime. Please note that when setting a smaller size then the current size, the eviction of data will be triggered which can take some time.
```